tools/downloader/pytorch_to_onnx.py

```diff
@@ -95,6 +104,7 @@ def convert_to_onnx(model, input_shape, output_file, input_names, output_names):
     output_file.parent.mkdir(parents=True, exist_ok=True)
     model.eval()
     dummy_input = torch.randn(input_shape)
+    model(dummy_input)
```

Comment:
What is the purpose of this?

Reply:
This fixes some conversion errors, when some layer's shape isn't defined explicitly by constant (_for some models_).

Reply:
OK, but could you add a comment explaining this? Otherwise, it just looks like debugging code that was left in.